profiles/Santosh1176.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/91916466/169870632-0ab75c94-7045-4aee-be36-6ecd734d06b4.png",
+  "issueId": 1131,
```

Comment:
```suggestion
  "issueId": 1130,
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix